src/exe-elf.c

```diff
@@ -378,5 +379,5 @@ void bxfi_lib_name_term(const char *str)
 
 size_t bxfi_exe_get_vmslide(bxfi_exe_lib lib)
 {
-    return lib->l_addr;
+    return (uintptr_t)lib->l_addr;
```

Comment:
I think you meant to cast to `size_t`. Although I think that the "most correct" type for bxfi_exe_get_vmslide should be `ptrdiff_t`, I guess. I'll address that in another commit.